src/extensions/extra/PDFView/styles/viewer-overrides.css

```diff
@@ -14,13 +14,67 @@ div.toolbar {
     display: none !important;
 }
 
-/** TODO: this needs some love **/
-.light-theme {
-    background-color: white;
-    color: black;
+body.dark-theme {
+    background: #222;
 }
 
-.dark-theme {
-    background-color: black;
-    color: white;
+body {
+    background: white;
+}
+
+body.dark-theme .meta, body.dark-theme .meta strong {
+    color: rgba(255,255,255,.6);
+}
+
+.meta, .meta strong {
+    color: rgba(0,0,0,.7);
+}
+
+
+#pdf-page-count {
+    position: relative;
+    padding-right: 16px;
+}
+
+#pdf-page-count:after {
+    content: "";
+    position: absolute;
+    right: 2px;
+    top: 0;
+    display: block;
+    background-image: url("/src/styles/images/icons/file-black.svg");
```

Comment:
We can't do this, since we also ship from `/dist`.  What you need to do is put the icons within this extension's dir, create an `icons/` dir or something.